<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:rgb(255, 255, 255);
font-size:15px;
font-style:MingLiU-ExtB;
}
QFrame{
border:sold 10px rgba(0,0,0);
background-image: url(:/assets/loginbackground.png);
}
QLineEdit{
color:#8d98a1;
background-color:rgba(64, 83, 97, 230);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QLineEdit:hover{
color:#8d98a1;
background-color:rgba(64, 83, 97, 80);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QLineEdit:focus{
color:#8d98a1;
background-color:rgba(64, 83, 97, 180);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}

QPushButton{
background:rgb(65, 175, 235);
color:rgb(220, 220, 220);
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton:hover{
background-color:rgba(80, 190, 250,130);
border-style:inset;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color:rgba(80, 190, 250,255);
border-style:inset;
font-style:MingLiU-ExtB;
}

QPushButton#hideButton{
background: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#hideButton:hover{
background: rgba(212, 212, 212, 100);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#hideButton:pressed{
background: rgba(212, 212, 212, 255);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#closeButton{
background: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#closeButton:hover{
background: rgba(255, 0, 0, 100);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#closeButton:pressed{
background: rgba(255, 0, 0, 255);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string extracomment="无功能label，可以换成其他字或者图片">WeNoChat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>652</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hideButton">
    <property name="geometry">
     <rect>
      <x>622</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>—</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
background-color: rgb(255, 255, 255,0);
font-style:MingLiU-ExtB;
}
QPushButton:hover{
background-color:rgba(220, 220, 220, 50);
border-style:inset;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color:rgba(80, 190, 250,130);
border-style:inset;
font-style:MingLiU-ExtB;
}</string>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forgetButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>390</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
background-color: rgb(255, 255, 255,0);
font-style:MingLiU-ExtB;
}
QPushButton:hover{
background-color:rgba(220, 220, 220, 50);
border-style:inset;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color:rgba(80, 190, 250,130);
border-style:inset;
font-style:MingLiU-ExtB;
}</string>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>registButton</tabstop>
  <tabstop>forgetButton</tabstop>
  <tabstop>hideButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
